rfcs/wpt_live.md

```diff
@@ -0,0 +1,71 @@
+# RFC 36: wpt.live
+
+## Summary
+
+Maintain a publicly-accessible deployment of the web-platform-tests project,
+hosted at http://wpt.live. Maintain another deployment which allows project
+contributors to preview their submissions during the peer review process,
+hosted at http://wptpr.live.
+
+## Details
+
+Bocoup and Google have collaborated on a service to provide a
+publicly-accessible instance of the web-platform-tests. The source code and the
+infrastructure configuration are maintained in the following repository:
+
+https://github.com/bocoup/web-platform-tests.live
```

Comment:
Is there a plan to move that under the wpt repo?

In general I'd like to see documentation here about what additional non-code pieces are in play for this new service (credentials, certtificates, etc.). It needs to be possible for people to get the access required to perform ongoing maintainance.

Reply:
> Is there a plan to move that under the wpt repo?

Yup; I've amended the RFC to state that

> In general I'd like to see documentation here about what additional non-code pieces are in play for this new service (credentials, certtificates, etc.). It needs to be possible for people to get the access required to perform ongoing maintainance.

Just access to the "wpt-live" project in Google Cloud Platform. I believe this is the extent of the secrets necessary to administer the project. Certificate renewal is automated via Let's Encrypt, so it doesn't represent a source of privileged information. I've updated the "risks" section with this information.